deploy/helm/metrics-discovery-config.yaml

```diff
@@ -0,0 +1,268 @@
+      max_interval: {{ .Values.aws_fargate.metrics.autodiscovery.retry_on_failure.max_interval }}
+      max_elapsed_time: {{ .Values.aws_fargate.metrics.autodiscovery.retry_on_failure.max_elapsed_time }}
+    sending_queue:
+      enabled: {{ .Values.aws_fargate.metrics.autodiscovery.sending_queue.enabled }}
+      num_consumers: {{ .Values.aws_fargate.metrics.autodiscovery.sending_queue.num_consumers }}
+      queue_size: {{ .Values.aws_fargate.metrics.autodiscovery.sending_queue.queue_size }}
+{{- if .Values.aws_fargate.metrics.autodiscovery.sending_queue.offload_to_disk }}
+      storage: file_storage/sending_queue
+    timeout: {{ .Values.aws_fargate.metrics.autodiscovery.timeout }}
+{{- end }}
+extensions:
+{{- if .Values.aws_fargate.metrics.autodiscovery.sending_queue.offload_to_disk }}
+  file_storage/sending_queue:
+    directory: /var/lib/swo/sending_queue
+{{- end }}
+  health_check:
+    endpoint: 0.0.0.0:13133
+  k8s_observer:
+    auth_type: serviceAccount
+    observe_pods: true    
+
+processors:
+  swk8sattributes:
+{{ include "common.k8s-instrumentation" . | indent 4 }}
+
+  memory_limiter:
+{{ toYaml .Values.aws_fargate.metrics.autodiscovery.memory_limiter | indent 4 }}
+
+  transform/scope:
+    metric_statements:
+      - context: scope
+        statements:
+          - set(name, "")
+          - set(version, "")
+
+  metricstransform/rename/discovery:
+    transforms:
+      # add `k8s.` prefix to all metrics
+      - include: ^(.*)$$
+        match_type: regexp
+        action: update
+        new_name: {{ .Values.otel.metrics.autodiscovery.prefix }}$${1}
+
+ 
+  groupbyattrs/common-all:
+    keys:
+      - k8s.container.name
+      - k8s.namespace.name
+      - k8s.pod.name
+      - k8s.pod.uid
+      - k8s.node.name
+      - host.name
+      - service.name
+  
+  resource/all:
+    attributes:
+      # Collector and Manifest version
+      - key: sw.k8s.agent.manifest.version
+        value: ${MANIFEST_VERSION}
+        action: insert
+
+      - key: sw.k8s.agent.app.version
+        value: ${APP_VERSION}
+        action: insert
+
+      # Cluster
+      - key: sw.k8s.cluster.uid
+        value: ${CLUSTER_UID}
+        action: insert
+
+      - key: k8s.cluster.name
+        value: ${CLUSTER_NAME}
+        action: insert
+ 
+
+{{- if .Values.otel.metrics.filter }}
+  filter/metrics:
+    metrics:
+{{ toYaml .Values.otel.metrics.filter | indent 6 }}
+{{- end }}
+
+  batch/metrics:
+{{ toYaml .Values.aws_fargate.metrics.autodiscovery.batch | indent 4 }}
+  
+{{- include "common-config.filter-remove-temporary-metrics" . | nindent 2 }}
+
+  filter/histograms:
+    metrics:
+      metric:
+        - 'type == METRIC_DATA_TYPE_HISTOGRAM and not(name == "k8s.apiserver_request_duration_seconds" or name == "k8s.workqueue_queue_duration_seconds" or name == "k8s.coredns_dns_request_duration_seconds" or name == "k8s.coredns_dns_request_size_bytes" or name == "k8s.coredns_dns_response_size_bytes")'
```

Comment:
Are the allowed exceptions correct? The list used in autodiscovery is different.